assets/app/views/builds.html

```diff
@@ -143,11 +141,15 @@
             </dl>
           </div>
         </div>
-        <div class="well" style="margin-bottom: 10px;" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
+        <div class="well build-log" ng-repeat="build in buildsByBuildConfig[buildConfigName] | orderObjectsByDate : true">
```

Comment:
**switched class names**
build-log to build-unit (felt that 'build' was to generic)
pod-log to pod-unit (felt that 'build' was to generic)
log-meta to build-detail and pod-detail
log-meta-title to build-detail-label and pod-detail-label